CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3100](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3100))
 - Add support to database stability opt-in in `_semconv` utilities and add tests
   ([#3111](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3111))
+- `opentelemetry-opentelemetry-pymongo` Add `py.typed` file to enable PEP 561
```

Comment:
This is jinja2, but to avoid a ton of conflicts or rebasing maybe we can add a cumulative entry after the pr has been merged?